pandas/core/indexes/base.py

```diff
@@ -958,6 +958,10 @@ def take(
         if kwargs:
             nv.validate_take((), kwargs)
         indices = ensure_platform_int(indices)
+
+        if indices.ndim == 0:
```

Comment:
how about `indices.ndim != 1`

Reply:
@jreback can you please help me with the whatsnew note? I do not know how & where to add it 😅